prediction_market_agent/tools/anvil/reset_balance_anvil.py

```diff
@@ -0,0 +1,30 @@
+import typer
```

Comment:
This script is duplicated in scripts/reset_balance_anvil.py

Reply:
removed